src/pages/studyView/charts/mutationPlot/StudyViewMutationMapperStore.ts

```diff
@@ -0,0 +1,114 @@
+import { Mutation } from 'cbioportal-ts-api-client';
+import {
+    ApplyFilterFn,
+    DataFilterType,
+    DefaultMutationMapperDataStore,
+    DefaultMutationMapperStore,
+    FilterApplier,
+    groupDataByGroupFilters,
+} from 'react-mutation-mapper';
+import { computed, makeObservable } from 'mobx';
+import _ from 'lodash';
+import {
+    createAnnotatedProteinImpactTypeFilter,
+    isPutativeDriver,
+} from 'shared/lib/MutationUtils';
+import { ProteinImpactType } from 'cbioportal-frontend-commons';
+
+export type SampleData = {
+    sampleId: string;
+    patientId: string;
+    studyId: string;
+    value: string;
+};
+
+interface DefaultMutationMapperStoreConfig {
```

Comment:
do we really need to duplicate `DefaultMutationMapperStoreConfig`? We can probably reuse the existing one.

Reply:
Here, the package does not have `DefaultMutationMapperStoreConfig` exported, so needed to create a copy on our frontend app.

Reply:
We can export `DefaultMutationMapperStoreConfig` from the package's `index.tsx`. We already made some modifications to the package itself, so it should okay to modify the `index.tsx` as well.

Reply:
Done!